packages/compass-explain-plan/src/components/explain-tree/explain-tree.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useRef, useEffect } from 'react';
+import { map } from 'lodash';
+import d3 from 'd3';
+import { ExplainStage } from '../explain-stage';
+import {
+  css,
+  palette,
+  spacing,
+  useDarkMode,
+} from '@mongodb-js/compass-components';
+
+import STAGE_CARD_PROPERTIES from '../../constants/stage-card-properties';
+
+interface ExplainTreeProps {
+  nodes: any[];
+  links: any[];
```

Comment:
Is there any chance we can improve these?

Reply:
absolutely I have a follow pr where I've typed everything